### ISSUE TITLE
get_helm.sh doesn't work with openssl > 1.0.1

### ISSUE THREAD

**Issue by mtricolici** [kubernetes/helm#2859]
```
$cat Dockerfile 
FROM debian:9

RUN apt-get update -q && \
  apt-get install -yq curl openssl && \
  dpkg-query -l|grep ssl && \
  curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash

$docker build -t qqq .
....

ii  libssl1.0.2:amd64         1.0.2l-2                       amd64        Secure Sockets Layer toolkit - shared libraries
ii  libssl1.1:amd64           1.1.0f-3                       amd64        Secure Sockets Layer toolkit - shared libraries
ii  openssl                   1.1.0f-3                       amd64        Secure Sockets Layer toolkit - cryptographic utility

...
Downloading https://kubernetes-helm.storage.googleapis.com/helm-v2.6.0-linux-amd64.tar.gz
Invalid command 'sha'; type "help" for a list.
```

```
$cat Dockerfile 
FROM debian:8

RUN apt-get update -q && \
  apt-get install -yq curl openssl && \
  dpkg-query -l|grep ssl && \
  curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
...

ii  libssl1.0.0:amd64         1.0.1t-1+deb8u6                  amd64        Secure Sockets Layer toolkit - shared libraries
ii  openssl                   1.0.1t-1+deb8u6                  amd64        Secure Sockets Layer toolkit - cryptographic utility

Successfully built 0b57aaae7c56
Successfully tagged qqq:latest
```

**Comment by jascott1** [comment]
@mtricolici Thanks for creating this issue. I think I will take a quick stab at resolving this since its a problem in most recent Debian image. Until a fix lands you should be able to edit the `get` script and change `sha` to `sha1` as a workaround. Something like
```
RUN curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get -O get 
&& sed s/sha\s/sha1\s/g get > get.fixed
&& ./get.fixed
```
Just some quick pseudo docker to express the idea. Im sure the sed and curl need work.

**Comment by thomastaylor312** [comment]
This was closed in #2860
